api/v2/db/ldap/connection.js

```diff
@@ -0,0 +1,28 @@
+const config = require('config');
+const ldap = require('ldapjs');
+
+const ldapConfig = config.get('dataSources').ldap;
```

Comment:
The better practice to get a nested config object with [config module](https://www.npmjs.com/package/config#quick-start) is like:

```js
config.get('dataSources.ldap');
```

example: https://github.com/osu-mist/notes-api/blob/master/api/v1/db/json/fs-operations.js#L4